packages/spec/src/spec.emu.html

```diff
@@ -372,7 +372,7 @@ <h1>Syntactic Grammar</h1>
     `...` ReferenceExpression
 
 InterfaceStatement :
-    `interface` Identifier TemplateParameters? InterfaceHeritage? `{` InterfaceBody? `}`
+    DecoratorList? `interface` Identifier TemplateParameters? InterfaceHeritage? `{` InterfaceBody? `}`
```

Comment:
I see some other cases in here that could also have `DecoratorList?` applied before them, but let's go ahead and add this one.

Just want to confer with @bterlson -- is this grammar file used for anything?

Reply:
Confirmed that we're not generating any code or anything from this document.

Reply:
I recognized that outsiders use it as a document to check the specs.
For example, tree-sitter.